app-devel/judy/autobuild/patches/01_notrans_man.patch

```diff
@@ -0,0 +1,9 @@
+diff -urpN ori/doc/Makefile.am new/doc/Makefile.am
```

Comment:
File names are not complaint with the Styling Manual, the use of AOSC-Tracking is recommended.

Reply:
(In which case, please mark the sources for each patch - which I assume to be Debian judging from the file names)

Reply:
Patches are maintained in AOSC Tracking now.
https://github.com/AOSC-Tracking/judy